internal/pkg/workspace/errors.go

```diff
@@ -32,18 +32,30 @@ func (e *ErrFileNotExists) Error() string {
 	return fmt.Sprintf("file %s does not exists", e.FileName)
 }
 
+// ErrTargetNotFound means that we couldn't locate the target file or the target directory.
+type ErrTargetNotFound struct {
```

Comment:
Why do we need an additional error type 🤔 Is it because `ErrWorkspaceNotFound` is specifically for directory?

Reply:
> Is it because ErrWorkspaceNotFound is specifically for directory?

This is partially the reason. It is mostly because `ErrWorkspaceNotFound` has `RecommendedActions` that's very specific.